wiki/de/de-Command-Line-Options.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: wiki
+title: "Kommandozeilen Optionen"
+lang: "en"
+permalink: "/wiki/Command-Line-Options"
+---
+
+# Kommandozeilen Optionen
+
+Zusätzlich zum setzen der Funktionen in der Jamulus Oberfläche kann man auch Funktionen über die Kommandozeile setzen. Die genaue Vorgehensweise hängt vom verwendeten Betriebssystem ab.
+
+Um zum Beispiel bei Windows die Rekorderfunktion mit dem `-R` Argument zu aktivieren : Rechtsklick auf die Jamulus Verknüpfung und wählen Sie "Eigenschaften" > Ziel. Füge dann die nötigen Argumente zu Jamulus.exe hinzu:
+
+`"C:\Program Files (x86)\Jamulus\Jamulus.exe" -s -R "C:\path\to\your\Jam Recordings"`
+
+Bei MacOS, starte das Terminal Fenster und führe Jamulus mit den gewünschten Optionen wie folgt aus:
+
+`/Applications/Jamulus.app/Contents/MacOS/Jamulus -s -R "/path/to/your/jam recordings"`
+
+***
+
+Sie können alle verfügbaren Optionen zur aktuellen Version von Jamulus über den Parameter `-h` erfahren.
```

Comment:
I used "du", not "Sie" since I think it sounds better for a software like Jamulus.